.github/workflows/efiling-fe-jest-cc.yml

```diff
@@ -1,4 +1,4 @@
-name: build
+name: Yarn - Frontend Jest Tests
```

Comment:
Let's name this:
```
efiling-frontend Jest Tests and Coverage
```

Reply:
cause there will be a separate one for demo app too

Reply:
should be done now, check it